about.html

```diff
@@ -12,53 +12,84 @@
 </head>
 
 <body>
+
+  <section class="top-of-page" alt="You find the nav bar and stuff here"><!--A new container to center the top of the page with the rest -->
+
+  <header>
+
+  <a href="index.html"><img src="img/lambda-black.png" alt="lambda logo in black"></a>
+
+  <nav>
+
+  <a href="index.html">Home</a>
+  <a href="#">About</a>
+  <a href="#">Products</a>
+  <a href="#">Blog</a>
+  <a href="#">Contact</a>
+
+  </nav>
+
+  </header>
+
+  <img src="img/jumbo-about.png" alt="A really big pic" class="large-img">
+
+  </section><!--End of 'top-of-page' class -->
+
     <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <div class="about-page-intro"><!--for styling this one little section-->
+    <h1>About</h1>
 
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </div>
 
+    <section alt="A bunch of stuff to learn more about" class="mid-section"><!--New container to style mid section -->
+
```

Comment:
the alt attributes go on image tags, you can remove that from the section tag here. also, on each page you can add one <main> </main> tag around the most pertinent info in place of a section tag. this will help with screen readers for those that are visually impaired